twill-core/src/main/java/org/apache/twill/internal/EnvKeys.java

```diff
@@ -22,36 +22,25 @@
  */
 public final class EnvKeys {
 
-  public static final String TWILL_ZK_CONNECT = "TWILL_ZK_CONNECT";
-  public static final String TWILL_APP_RUN_ID = "TWILL_APP_RUN_ID";
   public static final String TWILL_RUN_ID = "TWILL_RUN_ID";
```

Comment:
This key is to provide runId for TwillContainerService, which is not the application id.